tests/acceptance/features/apiAccountsHashDifficulty/createShareToSharesFolder.feature

```diff
@@ -1,23 +0,0 @@
-@skipOnReva
-Feature: sharing
```

Comment:
why are we deleting this feature file? :thinking:

Reply:
We have a lot of same tests. for example here `tests/acceptance/features/coreApiShareManagementToShares/acceptShares.feature` 
I deleted it how duplicate

Reply:
But this was only basic share test for special case i.e. running tests with default account-hash-difficulty